java/build.gradle

```diff
@@ -0,0 +1,117 @@
+plugins {
+	id 'java'
+	id 'com.diffplug.gradle.spotless' version '3.26.0'
+}
+
+allprojects {
+	repositories {
+		mavenCentral()
+	}
+}
+
+java {
+	sourceCompatibility = JavaVersion.VERSION_1_8
+	targetCompatibility = JavaVersion.VERSION_1_8
+	withJavadocJar()
+	withSourcesJar()
+}
+
+wrapper {
+	gradleVersion = '6.1.1'
+}
+
+spotless {
+	//java {
+	//	removeUnusedImports()
+	//	googleJavaFormat()
+	//}
```

Comment:
Should probably either uncomment or remove?

Reply:
this was originally in this pr and will be in the PR I open immediately after this one is merged